### PR TITLE
Event ordering

### PR DESCRIPTION
**Pull request by LudvigHz**
This adds optional ordering to the event view via url params.

### CODE REVIEW

File: lego/apps/events/filters.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from django_filters import CharFilter, DateFilter, FilterSet
+from django_filters import CharFilter, DateFilter, FilterSet, OrderingFilter
```
Comment (odinuge) [inline lego/apps/events/filters.py]: Do you need this?

File: lego/apps/events/views.py [hunk 2]
```diff
@@ -57,6 +59,12 @@
 class EventViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
 
     filter_class = EventsFilterSet
+    filter_backends = (
+        DjangoFilterBackend,
+        filters.OrderingFilter,
+        LegoPermissionFilter,
+    )
+    ordering_fields = ("start_time", "end_time", "title", "registration_deadline_hours")
```
Comment (odinuge) [inline lego/apps/events/views.py]: I dont think we need `registration_deadline_hours`. It is only a fixed integer count of hours before the events starts, that the registration closes.

Comment (LudvigHz) [reply lego/apps/events/views.py]: I thought maybe you might want to use this in combination with the start time for sorting after when the registration closes. But maybe its not so useful?

File: lego/apps/events/filters.py (hunk 1 above)
Comment (LudvigHz) [reply lego/apps/events/filters.py]: nope, forgot to remove that

**Pull request merged**